cri-o.yml

```diff
@@ -169,6 +171,22 @@
         regexp: 'overlay2\.override_kernel_check=1'
         state: present
       when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
+    - block:
```

Comment:
This commit includes `group_vars/all` file which sets `runtime` variable to `runc` by default.

User can override that variable int he `group_vars/all` file, in the hosts file or in the cmdline